_source/logzio_collections/_metrics-sources/ec2.md

```diff
@@ -116,21 +130,174 @@ logzio/docker-collector-metrics
 {:.paramlist}
 
 
 ##### Monitor advanced EC2 metrics (_Optional_)
 
 If you want, you can also monitor advanced EC2 metrics, including disk memory and swap memory. 
 
-To enable this option, you'll need to install and configure a Cloudwatch agent on your machine and specify the **CWAgent** namespace.
+To enable this option, you’ll need to install and configure a CloudWatch agent on your machine and specify the **CWAgent** namespace under the AWS_NAMESPACES parameter. For example:
+
+```yml
+metricsets:
+    - cloudwatch
+  metrics: #specify aws namespaces you want to monitor, just add namspaces from AWS list
+    - namespace: AWS/EC2
+    - namespace: CWAgent
+```
 
-See the official instructions for [installing the CloudWatch agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Agent-on-EC2-Instance.html) and [configuring it](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-cloudwatch-agent-configuration-file-wizard.html).
+For additional instructions, see more about [installing the CloudWatch agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Agent-on-EC2-Instance.html) and [configuring it](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-cloudwatch-agent-configuration-file-wizard.html).
 
 
 {% include metric-shipping/open-dashboard.html title="Cloudwatch AWS/EC2" %}
 
+</div>
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="ec2-vanilla">
+
+You have the option to ship CloudWatch metrics directly over Metricbeat, without a Docker container.
+
+**Recommended version**: Metricbeat version 7.5.x
+
+#### Configuration
+
+{% include trust-chain-warning.html msg='docker' %}
+
+<div class="tasklist">
+
+##### Set up your IAM user
+
+You'll need an [IAM user](https://console.aws.amazon.com/iam/home)
+with these permissions:
+
+* `cloudwatch:GetMetricData`
+* `cloudwatch:ListMetrics`
+* `ec2:DescribeInstances`
+* `ec2:DescribeRegions`
+* `iam:ListAccountAliases`
+* `sts:GetCallerIdentity`
+
+If you don't have one, set that up now.
+
+Create an **Access key ID** and **Secret access key** for the IAM user,
+and paste them in your text editor.
+
+You'll need these for your Metricbeat configuration later.
+
+##### Get your metrics region
+
+You'll need to specify the AWS region you're collecting metrics from.
+
+![AWS region menu](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/region-menu.png)
+
+Find your region's slug in the region menu
+(in the top menu, on the right side).
+
+For example:
+The slug for US East (N. Virginia)
+is "us-east-1",
+and the slug for Canada (Central) is "ca-central-1".
+
+Paste your region slug in your text editor.
+You'll need this for your Metricbeat configuration later.
+
+##### Download the Logz.io public certificate to your Metricbeat server
```

Comment:
Not 100% sure if this section needs to change because of recent certificate events